src/supernode.cpp

```diff
@@ -32,6 +32,8 @@ namespace graft
 namespace snd
```

Comment:
It should be graft::supernode ?

Reply:
Didn't catch your idea.
namespace SND is not subject of this PR.
This PR only and only about graft::supernode::requst. And nothing more. All the rest will be changed during the following steps.
Did I reply on your question?